ui/src/api/index.js

```diff
@@ -42,7 +42,7 @@ export function api (command, args = {}, method = 'GET', data = {}) {
 export function login (arg) {
   const params = new URLSearchParams()
   params.append('command', 'login')
-  params.append('username', arg.username)
+  params.append('username', arg.username || arg.email)
```

Comment:
The culprit is in Login.vue, but this is okay.

Reply:
https://github.com/apache/cloudstack/blob/master/ui/src/views/auth/Login.vue#L182